mlflow/R/mlflow/R/model-flavor.R

```diff
@@ -30,16 +11,18 @@ mlflow_load_flavor <- function(model_path) {
   UseMethod("mlflow_load_flavor")
 }
 
-#' Predict over MLflow Model Flavor
+#' Generate Prediction with MLflow Model
 #'
 #' Performs prediction over a model loaded using
 #' \code{mlflow_load_model()}, to be used by package authors
 #' to extend the supported MLflow models.
 #'
 #' @param model The loaded MLflow model flavor.
 #' @param data A data frame to perform scoring.
+#' @param ... Optional additional arguments passed to underlying predict
+#'   methods.
 #'
 #' @export
-mlflow_predict_flavor <- function(model, data) {
-  UseMethod("mlflow_predict_flavor")
+mlflow_predict <- function(model, data, ...) {
```

Comment:
nit: should we move this to model.R?
what about load_flaovr? should that be changed to load model in the same way?